src/libraries/Microsoft.PowerFx.Connectors/Internal/ConnectorErrors.cs

```diff
@@ -10,8 +12,8 @@ public ConnectorErrors()
         { 
         }
 
-        public ConnectorErrors(string error)
-            : base(error)
+        public ConnectorErrors(string error, ErrorResourceKey warning = default)
```

Comment:
>, ErrorResourceKey warning = default) [](http://example.com/codeflow?start=43&length=37)

Can we do in non-breaking way?

Public API changes:
❌CP0002 M:Microsoft.PowerFx.Connectors.SupportsConnectorErrors.#ctor(System.String)

Reply:
internal class, no one is using this class